src/components/uv-scroll.js

```diff
@@ -1,20 +1,79 @@
+const textureToData = new Map();
+const registeredTextures = [];
+
+export class UVScrollSystem {
+  tick(_t, dt) {
```

Comment:
Remove unused `_t`. 

Not sure if this was intentional (the underscore makes me think it was). If it was I'd argue we don't need/want to pass unused values here (or anywhere)